tests/pytest/conftest.py

```diff
@@ -35,6 +36,9 @@ def initialize_session(request):
     """Helper initializes a Django request object's session."""
 
     # https://stackoverflow.com/a/55530933/358804
-    middleware = SessionMiddleware(lambda x: x)
-    middleware.process_request(request)
+    middleware = [SessionMiddleware(lambda x: x), LocaleMiddleware(lambda x: x)]
```

Comment:
We should change the function name+docstring to reflect that it does more now.

Reply:
Done in [bb12659](https://github.com/cal-itp/benefits/pull/474/commits/bb12659b8be8436869c9c806851d75c29a459e35)